packages/devtools-modules/client/shared/components/splitter/SplitBox.js

```diff
@@ -60,6 +60,12 @@ const SplitBox = React.createClass({
     };
   },
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.vert !== nextProps.vert) {
+      this.setState({vert: nextProps.vert});
+    }
```

Comment:
it should also be possible to modify `width` and `height` (at least according to the comment 15 lines above). Not sure if I should add it now though, since I do not plan on using it.